src/Portfolio/Card.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace TeamWorkPm\Portfolio;
+
+use TeamWorkPm\Exception;
+use TeamWorkPm\Model;
+
+class Card extends Model
+{
+    public function init()
+    {
+        $this->parent = 'card';
+        $this->action = 'portfolio/cards';
+
+        $this->fields = [
+            'projectId' => [
+                'required' => false,
+                'attributes' => ['type' => 'string'],
+            ],
+
+            // These are only used by the update method
+            'cardId' => [
+                'required' => false,
+                'sibling' => true,
+                'attributes' => ['type' => 'string'],
+            ],
+
+            'columnId' => [
+                'required' => false,
+                'sibling' => true, // Dont nest under 'Card'
+                'attributes' => ['type' => 'string'],
+            ],
+
+            'oldColumnId' => [
+                'required' => false,
+                'sibling' => true, // Dont nest under 'Card'
+                'attributes' => ['type' => 'string'],
+            ],
+
+            'positionAfterId' => [
+                'required' => false,
+                'sibling' => true, // Dont nest under 'Card'
+                'attributes' => ['type' => 'integer'],
+            ],
+        ];
+    }
+
+    /**
+     * Get all the Columns for a Portfolio Column
+     * GET /portfolio/columns/{columnId}/cards
+     *
+     * @param integer $columnId
+     *
+     * @return \TeamWorkPm\Response\Model
+     * @throws \TeamWorkPm\Exception
+     */
+    public function getAllForColumn($columnId)
+    {
+        // Doesnt work
```

Comment:
what exactly doesn't work? If it doesn't work, why is this method implemented?

Reply:
Whoops, this comment was left by mistake.

Added it as a not to myself that it wasn't returning cards, was going to point this out to TW, but upon further investigation, it needed these changes to return the data: https://github.com/loduis/teamwork.com-project-management/pull/45/files#diff-3d898e219b939f5a7a7f0f84ab366b7cR78

I'll remove this comment